lib/bearerstrategy.js

```diff
@@ -294,7 +294,7 @@ Strategy.prototype.authenticate = function authenticateStrategy(req) {
       var metadataURL = self._options.identityMetadata;
 
       if (self._options.policyName) {
-        log.info('B2C: using policy %s', self._options.policyName);
+      log.info('B2C: using policy %s', self._options.policyName);
```

Comment:
Why does the indentation look off here? Where are we closing the open bracket on the if line above?